backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -690,7 +691,7 @@ public PartialOktaUser findUser(String username) {
 
   @Override
   public String getApplicationStatusForHealthCheck() {
-    return app.getStatus().toString();
+    return Objects.requireNonNull(app.getStatus()).toString();
```

Comment:
if `app.getStatus()` returns null, would this then throw a null pointer instead of updating the Okta health?

Reply:
good catch! will update to account